src/systems/systems/src/hydraulic/aerodynamic_model.rs

```diff
@@ -0,0 +1,669 @@
+use nalgebra::{Rotation3, Vector3};
+
+use uom::si::{
+    area::square_meter, f64::*, force::newton, length::meter,
+    mass_density::kilogram_per_cubic_meter,
+};
+
+use crate::{shared::interpolation, simulation::UpdateContext};
+
+/// Projects a 3D box with 3D size along a projection vector.
+/// Outputs the projected area on the plane normal to projection vector
+fn area_projected(size: Vector3<Length>, projection_vector: Vector3<f64>) -> Area {
+    let norm_projection_vector = projection_vector.normalize();
+
+    Area::new::<square_meter>(
+        size[0].get::<meter>() * size[2].get::<meter>() * norm_projection_vector[1]
+            + size[0].get::<meter>() * size[1].get::<meter>() * norm_projection_vector[2]
+            + size[1].get::<meter>() * size[2].get::<meter>() * norm_projection_vector[0],
+    )
+}
+
+pub struct AerodynamicModel {
+    max_drag_normal: Vector3<f64>,
+    lift_axis: Vector3<f64>,
+    lift_normal: Vector3<f64>,
+
+    lift_area: Area,
```

Comment:
I personally think it would be good to have unit names in the variable name to be clear what it is. but maybe Rust covers that already

Reply:
We use a package that hides units indeed.  Then you just do lift_area.get::<square_meter>() or any other unit